eventhubs/data-plane/azure-eventhubs-eph/pom.xml

```diff
@@ -1,19 +1,23 @@
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>azure-eventhubs-eph</artifactId>
+
+    <name>Microsoft Azure SDK for Event Hubs Eph</name>
```

Comment:
What does Eph stand for - you should expand what it means and then in brackets have (EPH), e.g. "Microsoft Azure SDK for Event Hubs Extremely Phonetic Hydrogen (EPH)" - for whatever EPH actually stands for.

Reply:
EPH stands for "Event Processor Host"